src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer is working right now!");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped working!");
```

Comment:
Similarly, the stop message should also be more informative. Include the class name to indicate which machine has stopped working.